### PR TITLE
e.GetAll<>() now gets results across all ptypes

### PR DESCRIPTION
**Pull request by asutosh97**

### CODE REVIEW

File: management_api.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ func (e *Enforcer) GetAllNamedActions(ptype string) []string {
 
 // GetAllRoles gets the list of roles that show up in the current policy.
 func (e *Enforcer) GetAllRoles() []string {
-	return e.GetAllNamedRoles("g")
+	return e.model.GetValuesForFieldInPolicyAllPTypes("g", 1)
```
Comment (hsluoyz) [inline management_api.go]: This is not `PTypes`. Maybe rename it.

Comment (asutosh97) [reply management_api.go]: what shall be a suitable name?

Comment (hsluoyz) [reply management_api.go]: `GetValuesForFieldInPolicy` would be OK.

Comment (asutosh97) [reply management_api.go]: there is actually a function of that name, which returns values for a specific type.

**Pull request merged**